docs/BreakingChanges/069 EF no longer throws for QueryViews with specific characteristics.md

```diff
@@ -0,0 +1,26 @@
+## 69: EF no longer throws for QueryViews with specific characteristics
```

Comment:
{
  "Id": "69",
  "Title": "EF no longer throws for QueryViews with specific characteristics",
  "Details": "Entity Framework no longer throws a StackOverflowException exception when an app executes a query that involves a QueryView with a 0..1 navigation property that attempts to include the related entities as part of the query. For example, by calling `.Include(e => e.RelatedNavProp)`.",
  "Suggestion": "This change only affects code that uses QueryViews with 1-0..1 relationships when running queries that call .Include. It improves reliability and should be transparent to almost all apps. However, if it causes unexpected behavior, you can disable it by adding the following entry to the <appSettings> section of the app's configuration file:\n\n```xml\n<add key=\"EntityFramework_SimplifyUserSpecifiedViews\" value=\"false\" />\n```",
  "Link": "https://msdn.microsoft.com/en-us/library/dn720774(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.2",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis